Changelog.md

```diff
@@ -5,6 +5,7 @@
 ### 🚨 Breaking changes
 
 ### ✨ New features and improvements
+- Fix front-end Assignment view for students (#7533)
```

Comment:
The word "fix" is a bit misleading, as there wasn't an error before. You can use the verb "Improve" instead. Also, "front-end" and "view" are redundant; I think you can just say "Assignment view" here.